hadoop-hdds/pom.xml

```
@@ -131,6 +132,12 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
         <version>${hdds.version}</version>
       </dependency>
 
+    <dependency>
+      <groupId>org.apache.ozone</groupId>
+      <artifactId>RocksDBCheckpointDiffer</artifactId>
```

Leftover old artifact ID:

```suggestion
      <artifactId>rocksdb-checkpoint-differ</artifactId>
```

---

Thanks. Committed.